src/commands/app/init.js

```diff
@@ -108,7 +106,7 @@ class InitCommand extends BaseCommand {
     // 4. retrieve workspace details, defaults to Stage
     const workspace = await this.retrieveWorkspaceFromName(consoleCLI, org, project, flags.workspace)
     // 5. ask for exensionPoints, only allow selection for extensions that have services enabled in Org
-    const extensionPoints = await this.selectExtensionPoints(flags, orgSupportedServices)
+    const extensionPoints = await this.selectExtensionPoints(flags, orgSupportedServices, { login: true, orgId: org.id })
```

Comment:
In case we keep dynamic extension list, please split this into two methods:
`this.selectExtensionPointsStatic(flags, orgSupportedServices)`
`this.selectExtensionPointsDynamic(consoleCLI, flags, orgSupportedServices, orgId)`